charts/couchdb/values.yaml

```diff
@@ -193,7 +193,7 @@ couchdbConfig:
     require_valid_user: false
   couch_httpd_auth:
     # Setting default timeout to 365 days (365*24*60*60)
-    timeout: 31536000
+    # timeout: 31536000
```

Comment:
We should explicitly set a default value to use so that it is clear what the timeout value is and what the connection authentication timer is. I suggest using 20 min session timeout with a 10 minute session refresh timer. This will match the values used in the meep-session package.

Reply:
Changed the timeout to 20 min and refresh timer to 10 min